profiler/internal/pprofutils/delta_test.go

```diff
@@ -0,0 +1,94 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package pprofutils
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/matryer/is"
```

Comment:
Should we switch this to `assert` to match the rest of the repo? No need to drag in another dep, right?

Reply:
5e37504b960cbab725039df0345d3be24c2b9d56